.devcontainer/swift/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.236.0/containers/cpp/.devcontainer/base.Dockerfile
+
+# [Choice] Debian / Ubuntu version (use Debian 11, Ubuntu 18.04/22.04 on local arm64/Apple Silicon): debian-11, debian-10, ubuntu-22.04, ubuntu-20.04, ubuntu-18.04
+ARG VARIANT="bullseye"
```

Comment:
I'd argue that we should rather use a recent Ubuntu build `ubuntu-20.04` or even `ubuntu-22.04` for consistency with what we use elsewhere instead of debian, but I'm not feeling strongly about it.

Reply:
these first lines are copied from one of the default dockerfiles for codespaces, and is actually overridden in `devcontainer.json` to be `ubuntu-20.04` (for consistency with the semmle-code version, but I would not mind picking `ubuntu-22.04`. Still, there's probably no need to keep the genericity of the default dockerfile as I have modified it in any case, so I'll change this